packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -260,8 +285,6 @@ class AnalyzeCommand extends FlutterCommand {
 
     int exitCode = await process.exitCode;
 
-    host.deleteSync(recursive: true);
```

Comment:
Oh, looks like it's been promoted from a `[hint]` to an `[error]`, and that's why the regex isn't finding it.